build_npm.ts

```diff
@@ -7,7 +7,11 @@ await copy("testdata", "npm/umd/testdata", { overwrite: true });
 await copy("README.md", "npm/README.md", { overwrite: true });
 
 await build({
-  entryPoints: ["./main.ts"],
+  entryPoints: ["./lib.ts", {
+    kind: "bin",
+    name: "license_checker",
+    path: "./main.ts",
+  }],
```

Comment:
I've added this new `kind: "bin"` entrypoint. It creates a `"bin"` entry in package.json and adds `#!/usr/bin/env node\n` to the top of the referenced path as mentioned in https://docs.npmjs.com/cli/v7/configuring-npm/package-json#bin

Reply:
Nice! Great!